community/cypher/cypher-logical-plans-3.4/pom.xml

```diff
@@ -65,7 +65,7 @@
 
     <dependency>
       <groupId>org.neo4j</groupId>
-      <artifactId>neo4j-cypher-frontend-3.4</artifactId>
+      <artifactId>openCypher-frontend</artifactId>
```

Comment:
I'm guessing the removal of the version number from the module is because this frontend should only be used once within a single project (ie. unlike the cypher-compiler-x.y module we will not have multiple versions in a single Neo4j instance)?

However, doesn't the cypher-compiler-x.y depend on cypher-frontend-x.y, so we would still need versioned frontends for the versioned compilers (ie. when Neo4j includes two compilers, they each depend on a different frontend)?

Reply:
I'm not sure that `cypher-compiler-3.5` depending on `openCypher-frontend` version `1.0` and `cypher-compiler-4.0` depending on `openCypher-frontend` version `2.5` is a problem, is it? Will they conflict on the classpath?

We could of course redesign the compatibility layer so that Neo4j Cypher only depends on a single version of `openCypher-frontend` which is backwards compatible, and then have multiples of `cypher-compiler-x.y` to which it redirects based on compatibility mode selected by the user.